server/controllers/SigninOptions.js

```diff
@@ -0,0 +1,116 @@
+const db = require("../models");
+const SigninOption = db.SigninOption;
+const User = db.User;
+
+//These operations can be integrated into a LoginAuth controller
```

Comment:
@D-Garnes I dunno how this will integrate with your changes, ya'll will need to coordiate here.